app/lib/content_security_policy.rb

```diff
@@ -72,6 +72,13 @@ def s3_hostname_host
     host_to_url ENV.fetch('S3_HOSTNAME', nil)
   end
 
+  def swift_object_url_host
+    swift_object_url = ENV.fetch('SWIFT_OBJECT_URL', nil)
+    return if swift_object_url.blank?
+
+    host_to_url Addressable::URI.parse(swift_object_url).host
+  end
```

Comment:
Looks good, but couldn't we just use `SWITCH_OBJECT_URL` rather than just its host? CSP supports path prefixes, and the more precise we are, the more unlikely it is that someone will manage to include something unexpected.

Reply:
@ClearlyClaire Thank you for your review. I updated the code to use SWIFT_OBJECT_URL value instead of the host name.